plugins/polio/js/src/domains/Chronogram/Chronogram/index.tsx

```diff
@@ -22,16 +24,27 @@ export const Chronogram: FunctionComponent = () => {
     const classes: Record<string, string> = useStyles();
     const { formatMessage } = useSafeIntl();
 
+    const { data: chronogramMetaData, isFetching: isFetchingMetaData } =
+        useOptionChronogram();
+
     return (
         <>
             <TopBar
                 title={formatMessage(MESSAGES.chronogramTitle)}
                 displayBackButton={false}
             />
-            <Box className={classes.containerFullHeightNoTabPadded}>
-                <ChronogramFilters params={paramsNew} />
-                <ChronogramTable params={paramsNew} />
-            </Box>
+            {isFetchingMetaData && <LoadingSpinner />}
+            {!isFetchingMetaData && (
```

Comment:
It would be better to render the filters and pass `isFetchingMetaData` as `loading` to the `InputComponent` in your filters